CMakeLists.txt

```diff
@@ -157,6 +157,7 @@ add_library(semba-types
 	"src_main_pub/nfde_types.F90"
 	"src_main_pub/fdetypes.F90"
 	"src_mtln/mtln_types.F90"
+	"src_conformal/geometry.F90"

```

Comment:
This should be split into a conformal_types.F90 containing just types and and a geometry.F90 module with the needed logic